src/CMakeLists.txt

```diff
@@ -187,6 +181,13 @@ set(Mapper_Common_SRCS
  gui/symbol/symbol_setting_dialog.cpp
  gui/symbol/symbol_properties_widget.cpp
  gui/symbol/symbol_point_editor.cpp
+
+ gui/map/map_widget.cpp
+ gui/map/map_editor.cpp
+ gui/map/map_editor_activity.cpp
+ gui/map/map_dialog_new.cpp
+ gui/map/map_dialog_scale.cpp
+ gui/map/map_dialog_rotate.cpp
```

Comment:
I don't believe this is to many. To solve the `../` issue I think we should start using absolute paths (from `src`, so `"../../core/map_coord.h"` becomes `"core/map_coord.h"`, regardless of which file it is included from). If you are ok with that solution I can take on converting paths as part of this or a separate PR.